src/realm/sync/noinst/client_impl_base.hpp

```diff
@@ -570,6 +556,8 @@ class ClientImpl::Connection final : public WebSocketObserver {
     // At least one PING message was sent since connection was established
     bool m_ping_sent = false;
 
+    bool m_error_received = false;
```

Comment:
Can we rename this to something like "m_websocket_error_received" to differentiate it from receiving an in-band ERROR message?

Reply:
done